alvath96/cis-aes
# add AES encrypt & decrypt method without padding and CTR

Changes
---
 - add AES encrypt & decrypt method without padding and CTR
 - add multiplication & addition operation in Galois field methods

How to review
---
 - First, review Galois operations (`galoisMultiplication` and `galoisAddition` in `Utils.java`)
 - Check the constants, make sure they're all correct (`sbox`, `invSbox`, `invMixCol`, `invMixCol`)
 - Check functions for encryption (`subBytes`, `shiftRows`, `mixColumns`, `addRoundKey`)
 - Check functions for decryption (`invSubBytes`, `invShiftRows`, `invMixColumns`)
 - Check encryption & decryption logic

To Do
---
 - [ ] implement PKCS#5 padding
- [ ] implement CTR x AES
- [ ] use byte instead of hexadecimal string for input (plaintext & ciphertext)
